config/locales/en.yml

```diff
@@ -440,6 +440,7 @@ en:
       stop_using: Stop using this certificate
     confirmation:
       title: Confirmation
+      publish_failed_title: Publish Failed
```

Comment:
```suggestion
      publish_failed_title: Publish failed
```
I don't think we should capitalize titles

Reply:
👍 have amended.